c2cgeoform/views/abstract_views.py

```diff
@@ -306,7 +372,7 @@ def duplicate(self):
         self._populate_widgets(form.schema)
         rendered = form.render(dict_,
                                request=self._request,
-                               actions=self._item_actions())
+                               actions=self._item_actions(self._model()))
```

Comment:
why use self._model and not obj (cf edit)

Reply:
because src obj is already persitent and I want actions for a destination new one. I should use 'dest'.